.azure-pipelines.yml

```
@@ -76,6 +76,12 @@ jobs:
       testRunTitle: 'Publish test results for Python $(python.version)'
 
   - script: |
-      python setup.py check --restructuredtext --strict
       rst2html.py --halt=2 README.rst >/dev/null
     displayName: 'rst2html'
+
+  - script: |
+      pip install setuptools twine
+      python setup.py sdist bdist_wheel
```

Once this is merged, can you please add a PR that contains this change and targets the `flit` branch?

```suggestion
      pip install twine
      flit build
```

---

Guess I'll have to understand your choice of flit over alternatives first. But if the team decides that flit is the way to move forward instead of poetry, then I will be happy to submit the PR.